src/pineko/theory.py

```diff
@@ -378,18 +379,38 @@ def fk(self, name, grid_path, tcard, pdf):
         max_al = 0
         # check for sv
         if not np.isclose(xir, 1.0):
-            is_ren_as, is_ren_al = check.contains_ren(grid, max_as, max_al)
+            is_ren_as, is_ren_al, is_cen_as, is_cen_al = check.contains_ren(
+                grid, max_as, max_al
+            )
             if not (is_ren_as and is_ren_al):
                 raise ValueError(
                     "Renormalization scale variations are not available for this grid"
                 )
+            if is_ren_as and not is_cen_as:
+                rich.print(
```

Comment:
Prefer logging to printing.

Reply:
Yes but in this case there are really 3 different behaviour and I believe that we want to know if `is_ren_as` is `True` because really the ren_sv at that order are there or just because there is no central for that order (which is the case more or less always with NNLO)